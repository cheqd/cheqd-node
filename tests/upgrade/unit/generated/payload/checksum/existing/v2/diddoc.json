{
    "did_doc": {
        "context": [],
        "id": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
        "controller": [
            "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb"
        ],
        "authentication": [
            "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb#key-1"
        ],
        "assertion_method": [],
        "capability_invocation": [],
        "capability_delegation": [],
        "key_agreement": [],
        "also_known_as": [],
        "verification_method": [
            {
                "id": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
                "verification_material": "z6MkjndtVVvRivBfWddHt5QC2WHoguixzrVBxTTnCJW2ArYp"
            }
        ],
        "service": []
    },
    "metadata": {
        "created": "2021-09-14T13:00:00Z",
        "updated": "2021-09-14T13:00:00Z",
        "version_id": "75e0cf07-1729-40de-bd88-a4e091eff137",
        "deactivated": false,
        "next_version_id": "",
        "previous_version_id": ""
    }
}