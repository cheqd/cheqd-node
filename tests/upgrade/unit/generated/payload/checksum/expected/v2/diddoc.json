{
    "did_doc": {
        "context": [],
        "id": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
        "controller": [
            "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb"
        ],
        "authentication": [
            "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb#key-1"
        ],
        "assertionMethod": [],
        "capabilityInvocation": [],
        "capabilityDelegation": [],
        "keyAgreement": [],
        "alsoKnownAs": [],
        "verificationMethod": [
            {
                "id": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
                "verification_material": "{\"publicKeyMultibase\":\"z6LNquFfzPNhCQ8nbCWSMBQjosLT7ayEqGSYrN2Y1FdmS\"}"
            }
        ],
        "service": []
    },
    "metadata": {
        "created": "2021-09-14T13:00:00Z",
        "updated": "2021-09-14T13:00:00Z",
        "version_id": "75e0cf07-1729-40de-bd88-a4e091eff137",
        "deactivated": false,
        "next_version_id": "",
        "previous_version_id": ""
    }
}